coderedcms/models/snippet_models.py

```diff
@@ -216,6 +216,81 @@ def __str__(self):
         return "{0} > {1}".format(self.classifier.name, self.name)
 
 
+@register_snippet
+class FilmStrip(ClusterableModel):
+    class Meta:
+        verbose_name = _("Film Strip")
+
+    name = models.CharField(
+        max_length=255,
+        verbose_name=_("Name"),
+    )
+    offset = models.IntegerField(default=0)
+
+    panels = [
+        MultiFieldPanel(
+            heading=_("Film Strip"),
+            children=[
+                FieldPanel("name"),
+                FieldPanel("offset"),
+            ],
+        ),
+        InlinePanel("film_panels", label=_("Panels")),
+    ]
+
+    def __str__(self):
+        return self.name
+
+
+class FilmPanel(Orderable, models.Model):
+    class Meta:
+        verbose_name = _("Film Panel")
+
+    film_strip = ParentalKey(
+        FilmStrip,
+        related_name="film_panels",
+        verbose_name=_("Film Panel"),
+    )
+    image = models.ForeignKey(
```

Comment:
Change this to "background image" (use same wording / help text as hero unit block)